moonbeam-types-bundle/index.ts

```diff
@@ -61,6 +61,21 @@ export const rpcDefinitions: Record<string, Record<string, DefinitionRpc | Defin
       type: "Result<Vec<TransactionTrace>>",
     },
   },
+  xcm: {
+    injectDownwardMessage: {
+      description: "Inject a downward message from the relay chain.",
+      params: [{ name: "message", type: "Vec<u8>" }],
+      type: "Result<()>",
+    },
+    injectHrmpMessage: {
+      description: "Inject an HRMP message from a dedicated channel from a sibling parachain",
+      params: [
+        { name: "sender", type: "ParaId" },
+        { name: "message", type: "Vec<u8>" },
+      ],
+      type: "Result<()>",
+    },
+  },
```

Comment:
Was this addition necessary (i.e. was something not working without it?)

Reply:
Declaring RPC endpoints in the types bundle is necessary to be able to use those RPCs from Apps -> Developer -> RPC Calls. I made this change so that the RPCs appear in Apps in the future. But this fix will not help until it is merged and published all the way into Apps (unless I run Apps locally myself and use this updated types bundle of course).

Reply:
Also, I know we're still considering what to do with this part of the types bundle, and if we decide to just deprecate it, I'm totally fine with that, and this part can be removed. It's only a small convenience for me to use Apps.

But as long as we have it in the code I figure I might as well keep it up to date.

Reply:
Sounds good.